WBooks/Book_Information/BookInformationViewController.swift

```diff
@@ -9,8 +9,40 @@
 import UIKit
 import ReactiveSwift
 
+enum BookStatus: String {
+    case avaliable = "Avaliable"
+    case notAvaliable = "Not Avaliable"
+    
+    var textColor: UIColor {
+        switch self {
+        case .avaliable:
+            return UIColor(hex: "A5CD39")!
+        case .notAvaliable:
+            return UIColor(hex: "D0021B")!
+        }
+    }
+    
+    var rentEnabled: Bool {
+        switch self {
+        case .avaliable:
+            return true
+        case .notAvaliable:
+            return false
+        }
+    }
+}
+
 class BookInformationViewController: UIViewController {
     
+    // MARK: - Constants
+    
+    struct Constants {
+        static let addToWishlisSuccessMessage = "Book added to wishlist"
+        static let addToWishlisFailureMessage = "Book not added to wishlist. Try again"
+        static let rentBookSuccessMessage = "Book rented"
+        static let rentBookFailureMessage = "The book could not be rented. Try again"
```

Comment:
You can add a card, for the future, about `Localization` of strings, `Internationalization` of string. To investigate about it, and change all strings in the app to that. But again, in the future, just add a card in backlog.